projects/Mallard/src/screens/issue-screen.tsx

```diff
@@ -136,8 +140,23 @@ const IssueScreenWithPath = ({ path }: { path: PathToIssue | undefined }) => {
                         </FlexCenter>
                     </>
                 ),
-                success: issue => (
+                success: (issue, { retry }) => (
                     <>
+                        {preview && (
+                            <Button
+                                onPress={() => {
+                                    clearCache()
+                                    retry()
+                                }}
+                                buttonStyles={{
+                                    borderRadius: 0,
+                                    backgroundColor:
+                                        color.palette.lifestyle.bright,
+                                }}
+                            >
+                                (🗞🗑) → ⏰ → 📰
```

Comment:
as much as i love this I'd try to chuck this on the left of the navbar with the refresh glyph or, like this but with some accessibility text. I'd also keep the button shape and colors but wrap it in a container view so it reads as a button